react/features/virtual-background/reducer.js

```diff
@@ -20,13 +43,18 @@ ReducerRegistry.register('features/virtual-background', (state = {}, action) =>
 
     switch (action.type) {
     case SET_VIRTUAL_BACKGROUND: {
+        virtualBackground.virtualSource = virtualSource;
```

Comment:
This is unnecessary because the persistence layer looks at the redux tree itself, the object passed to the register() function is just for configuring what parts to persist. But as I mentioned earlier you can ignore the object and pass `true` since you want the entire state to be persisted.